docs/developers-clear-containers-install.md

```diff
@@ -124,7 +124,7 @@ $ sudo mkdir -p /etc/systemd/system/docker.service.d/
 $ cat << EOF | sudo tee /etc/systemd/system/docker.service.d/clear-containers.conf
 [Service]
 ExecStart=
-ExecStart=/usr/bin/dockerd -D --add-runtime clearcontainers=/usr/local/bin/cc-runtime --default-runtime=runc
+ExecStart=/usr/bin/dockerd -D --add-runtime cc-runtime=/usr/local/bin/cc-runtime --default-runtime=runc
```

Comment:
just to check, does this also align with what we do in our (OBS) packaging? @gorozco1 , care to comment?